website/content/en/docs/faq.md

```diff
@@ -0,0 +1,50 @@
+<!-- todo defaults+overrides -->
+### Does Karpenter support taints?
+Yes. Taints are an opt-out mechanism which allows users to specify the nodes on which a pod cannot be scheduled. Unlike node selectors, Karpenter does not automatically taint nodes in response to pod tolerations. Similar to node selectors, users may specify taints on their Provisioner, which will be automatically added to every node it provisions. This means that if a Provisioner is configured with taints, any incoming pods will not be scheduled unless the taints are tolerated.
+### Does Karpenter support topology spread constraints?
+Not yet. Karpenter plans to respect `pod.spec.topologySpreadConstraints` by v0.4.0.
+### Does Karpenter support node affinity?
+Not yet. Karpenter plans to respect `pod.spec.nodeAffinity` by v0.4.0.
+### Does Karpenter support custom resource like accelerators or HPC?
+Yes. Support for specific custom resources may be implemented by cloud providers. The AWS Cloud Provider supports `nvidia.com/gpu`, `amd.com/gpu`, `aws.amazon.com/neuron`.
+### Does Karpenter support daemonsets?
+Yes. Karpenter factors in daemonset overhead into all provisioning calculations. Daemonsets are only included in calculations if their scheduling constraints are applicable to the provisoned node.
+### Does Karpenter support multiple Provisioners?
+Each Provisioner is capable of defining heterogenous nodes across multiple availability zones, instance types, and capacity types. This flexibility reduces the need for a large number of Provisioners. However, users may find multiple Provisioners to be useful for more advanced use cases, such as defining multiple sets of provisioning defaults in a single cluster.
+### If multiple Provisioners are defined, which will my pod use?
+By default, pods will use the rules defined by a Provisioner named `default`. This is analogous to the `default` scheduler. To select an alternative provisioner, use the node selector `karpenter.sh/provisioner-name: alternative-provisioner`. You must either define a default provisioner or explicitly specify `karpenter.sh/provisioner-name` node selector.
+## Deprovisioning
+### How does Karpenter decide which nodes it can terminate?
+Karpenter will only terminate nodes that it manages. Nodes will be considered for termination due to expiry or emptiness (see below).
+### When does Karpenter terminate empty nodes?
+Nodes are considered empty when they do not have any pods scheduled to them. Daemonsets pods and Failed pods are ignored. Karpenter will send a deletion request to the Kubernetes API, and graceful termination will be handled by termination finalizer. Karpenter will wait for the duration of `ttlSecondsAfterUnderutilized` to terminate an empty node. If `ttlSecondsAfterUnderutilized` is unset, **which it is by default**, Karpenter will not terminate nodes once they are empty.
```

Comment:
The flag is currently `ttlSecondsAfterEmpty`

Reply:
@mikesir87 Thanks. I'm going to reworking the whole FAQ, but I'll make sure I use the latest value if I keep this section.

Reply:
I just did an update of the FAQ and I'm planning to move the section on deleting nodes to the [Deprovisioning nodes](https://karpenter.sh/docs/tasks/deprov-nodes/) section of the docs. For now, I'm just going to point to that section from the FAQ.